app/assets/stylesheets/app/loading.scss

```diff
@@ -0,0 +1,15 @@
+.loading-icon {
+  animation: spin 2s linear infinite;
+  border: 6px solid $gray; /* Light grey */
+  border-top: 6px solid $primary-color; /* Blue */
```

Comment:
Properties should be ordered animation, border, border-radius, border-top, height, margin, padding, width<br>Use `//` comments everywhere